lib/middleware/routes-static/routes-static.js

```diff
@@ -36,7 +33,7 @@ async function init (options = {}) {
     let isDirectory
 
     try {
-      const resourceDetails = await stat(resource)
+      const resourceDetails = statSync(resource)
```

Comment:
👍 and for the de-asyncification

Reply:
:D curious..why aren't you keen on it?

Reply:
Well as it's part of the start up and it is blocking, there's no reason for it not use the sync version. I just think I went a bit async-happy…